.github/workflows/c-cpp.yml

```diff
@@ -21,22 +21,26 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
-    container: niuslar/stm32-cubeide-headless:latest
+    container: 
+      image: ghcr.io/niuslar/stm32-cubeide-headless:latest
```

Comment:
In the workflow to build and push the image manually, the image name will vary depending on the username. What happens if someone else runs that workflow?